libs/ngtcp2/Makefile

```diff
@@ -1,12 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ngtcp2
-PKG_VERSION:=1.1.0
-PKG_RELEASE:=1
+PKG_VERSION:=1.4.0
+PKG_RELEASE:=r1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/ngtcp2/ngtcp2/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=987d784643edea4f2859c405f7dfbc53871a9f7ae5fcddf5fb12ec5dfce1ef22
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
```

Comment:
This was changed, but your commit description does not say why suddently you decided to use .xz instead of .gz.

Reply:
> @stangri marked this conversation as resolved. 
<img width="851" alt="image" src="https://github.com/openwrt/packages/assets/4096468/ead9f9cc-8e00-4bd1-9842-281e859a7c2e">

How? There wasn't any change, you have not add any description.